shared/src/commonMain/kotlin/ui/viewmodels/authentication/PresentationStateModel.kt

```diff
@@ -160,6 +163,14 @@ class PresentationStateModel {
         }
     }
 
+    /**
+     * Returns true if the device is not yet connected to the transport method.
+     */
+    fun isNotYetConnected(): Boolean {
+        return state.value == State.IDLE || state.value == State.INITIALISING
+                || state.value == State.CHECK_PERMISSIONS || state.value == State.NO_PERMISSION
+                || state.value == State.CONNECTING
+    }
```

Comment:
Can we add some debug output here that prints the actual state, when this is called